src/components/shared/Popover.js

```diff
@@ -1,12 +1,25 @@
-import { DOM as dom, PropTypes, createFactory, Component } from "react";
+/* @flow */
+import { DOM as dom, createFactory, Component } from "react";
 import ReactDOM from "../../../node_modules/react-dom/dist/react-dom";
 import classNames from "classnames";
 import _BracketArrow from "./BracketArrow";
 const BracketArrow = createFactory(_BracketArrow);
 
 import "./Popover.css";
 
+export type Props = {
+  target?: Object,
+  children?: Object,
+  onMouseLeave?: Function,
+  type?: string
+};
```

Comment:
I think the codemod added that; I just removed `export` and updated this PR.

What about all the flow errors?  Is that going down a rabbit hole trying to resolve them all?